ods_ci/tests/Tests/1000__model_serving/1009__model_serving_triton_on_kserve/1009__model_serving_triton_on_kserve.robot

```diff
@@ -35,10 +35,12 @@ ${ONNX_RUNTIME_FILEPATH}=    ${RESOURCES_DIRPATH}/triton_onnx_rest_servingruntim
 ${EXPECTED_INFERENCE_REST_OUTPUT_FILE}=      tests/Resources/Files/triton/kserve-triton-onnx-rest-output.json
 ${PATTERN}=  https:\/\/([^\/:]+)
 ${INFERENCE_REST_INPUT_PYTORCH}=    @tests/Resources/Files/triton/kserve-triton-resnet-rest-input.json
+${INFERENCE_GRPC_INPUT_PYTORCH}=    tests/Resources/Files/triton/kserve-triton-pytorch-gRPC-input.json
```

Comment:
is it same model as the REST one? (resnet?) - if yes, could you rename this file to contain the model name?

Reply:
will change berto resnet is the model name